css-canvas-medical/styles.css

```diff
@@ -0,0 +1,197 @@
+#privacy {
+  margin-right: 40px;
+}
+
+#end {
+  padding: 100px 10%;
+}
+
+.indented {
+  padding-left: 10%;
+  padding-right: 10%;
+}
+
+.relative {
+  position: relative;
+}
+
+.absolute {
+  position: absolute;
+}
+
+.border-bottom {
+  border-bottom: 1px solid rgb(231, 233, 236);
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+h2 {
+  font-family: 'Tiempos Headline', serif;
+}
+
+h3 {
+  font-family: 'Tiempos Headline', serif;
+  font-weight: 300;
+  margin: 0;
+}
+
+.blue-button {
+  background-color: hsl(213deg, 42%, 28%);
+  padding: 20px 50px;
+  border-radius: 4px;
+}
+
+.blue-button:hover {
+  background-color: hsl(213deg, 42%, 18%);
+  text-decoration: none;
+}
+
+.s-three .blue-button {
+  display: block;
+}
+
+.header-link-2 {
+  padding: 15px 17px;
+}
+
+.s-one {
+  margin-top: 35px;
+  margin-bottom: 35px;
+}
```

Comment:
Something like this could be reusable like the `px-something` to being `my-35` which would be margin along the y axis is 35 (something).

Just another naming convention note.